AU2/plugins/CorePlugin.py

```diff
@@ -523,10 +523,16 @@ def gather_config_names(self):
         Gathers the name of all ConfigExports from all plugins
         """
         names = []
-        for p in PLUGINS:
-            for c in p.config_exports:
-                names.append(c.display_name)
-        names.sort()
+        configs = (c for p in PLUGINS for c in p.config_exports)
+        for c in sorted(configs, key=lambda e: e.display_name):
+            disp = c.display_name
+            # display dangerous config options in red if the game has already started
+            if isinstance(c, DangerousConfigExport) and get_game_start() <= get_now_dt():
+                # this is kind of cheating because it's not really html compatible
+                # a `ansi2html` library does exist though,
+                # or we can create a specific HTMLcomponent to implement the colouring

```

Comment:
Let's make this an HTML component with a 'color' attribute (which is mapped at run-time from a string - e.g. "red" - to an implementation of red)

Reply:
I've just remembered why I did it like this in the first place[^1]. The list of plugin options is given using an `options_functions` rather than returning a `HTMLcomponent`. So having a HTMLcomponent would require a reworking of the plugin framework (say by having the `options_functions` return  HTMLcomponents rather than a list of options).
I think this would be reasonable to do (it would also allow us to extend #55 to `Assassin -> Update` for example), since I could have it check if the `options_functions` for a given export does return a HTMLcomponent and if not fall back to the current behaviour to avoid having to modify every single existing plugin export.

[^1]: (I started writing this code a couple days ago)

Reply:
Or, you add a meta component to implement the challenge, and then use `@property` to override accesses to underlying element and wrap it in the new meta component

Reply:
A simple improvement would be to return a list of `Options` for some new class `Option` which has associated metadata (such as color) and then create a "ColoredSelectorList". Idk. 

Or, have the front-end tell you whether it is HTML or not? Maybe you could find a good place for front-end to tell back-end its name.

Reply:
I'm also happy with the proposed changes you made (and would prefer those, if you have time)